client/pluginManger.go

```diff
@@ -68,6 +68,14 @@ type InstalledPlugin struct {
 	MinimumJavaVersion string
 	SupportDynamicLoad string
 	BackVersion        string
+	Dependencies       []Dependence
+}
+
+// Dependence represent the plugin's package dependence
+type Dependence struct {
```

Comment:
Suggest you change to`PluginDependency`.

Reply:
If there're two plugin dependency structs, it'll cause confuse too. We always should consider reuse instead of creating a new one. Besides, those two structs are pretty the same.